src/test/java/io/vertx/test/core/BufferTest.java

```diff
@@ -605,6 +605,7 @@ public void testGetBytes() throws Exception {
     Buffer b = Buffer.buffer(bytes);
 
     assertTrue(TestUtils.byteArraysEqual(bytes, b.getBytes()));
+    assertEquals(100, b.getBytes(0, 99).length);
```

Comment:
Just what the test case is asserting. I am expecting the length of the returned byte array to be 100.
When I read the parameter comments on [Buffer.getBytes(int, int)](https://github.com/eclipse/vert.x/blob/3.4.1/src/main/java/io/vertx/core/buffer/Buffer.java#L280) it indicates array positions. 0 being the first and 99 being the 100th.
Are you telling me a request for 100 elements in the array should only return 99 is expected ?